modules/install.am

```diff
@@ -1,320 +1,381 @@
-	METAPACKAGES="kdegames kdeutils libreoffice nodejs platform-tools"
-
-	for arg in $ARGS; do
-
-		function _install(){
-			echo ""
-			case $arg in
-
-			'--debug'|'--force-latest')	return 0;;
-			*) 
-				function _install_arg(){
-					chmod a+x ./$arg
-
-					# Check if the installation script is a metapackage (example one of the 40+ kdegames scripts)
-					for metapackage in $(echo "$METAPACKAGES"); do
-						if grep -q "$metapackage" ./$arg 2> /dev/null; then
-							if test -d $APPSPATH/$metapackage/tmp; then
-								$SUDOCOMMAND $APPSPATH/$metapackage/remove 2> /dev/null
-							elif test -d $APPSPATH/$metapackage; then
-								echo ' ◆ '$(echo '"'$arg'"' | tr a-z A-Z)' IS PART OF "'$(echo "$metapackage" | tr a-z A-Z)'", ALREADY INSTALLED'; return 0
-							fi
-						fi
-					done
-
-					# Prevent "appman" installing itself
-					if [ $arg == appman ]; then
-						if [ $AMCLI == appman ] 2>/dev/null; then
-							echo ' 💀 ERROR: YOU CANNOT INSTALL APPMAN USING APPMAN'; return 0
-						fi
-					fi
-
-					# Check if the installation script contain a keyword related to a missing dependence
-					if grep -q '.deb' ./$arg; then
-						if ! command -v ar 1>/dev/null; then
-							echo ' 💀 ERROR: YOU CANNOT INSTALL '$(echo '"'$arg'"' | tr a-z A-Z)' WITHOUT INSTALLING "BINUTILS"!'; return 0
-						fi
-					fi
-					if grep -q 'ffwa-' ./$arg; then
-						ffbrowser=$(find ${PATH//:/ } -maxdepth 1 -name firefox* | sort | head -1)
-						if [[ -z "$ffbrowser" ]]; then
-							echo ' 💀 ERROR: YOU CANNOT INSTALL '$(echo '"'$arg'"' | tr a-z A-Z)' WITHOUT INSTALLING "FIREFOX"!'; return 0
-						else
-							sed -i 's#firefox --class#'$(echo "$ffbrowser" | xargs -L 1 basename)' --class#g' ./$arg
-						fi
-					fi
-					if grep -q '.tar' ./$arg; then
-						if ! command -v tar 1>/dev/null; then
-							echo ' 💀 ERROR: YOU CANNOT INSTALL '$(echo '"'$arg'"' | tr a-z A-Z)' WITHOUT INSTALLING "TAR"!'; return 0
-						fi
-					fi
-					if grep -q '.zip' ./$arg; then
-						if ! command -v unzip 1>/dev/null; then
-							echo ' 💀 ERROR: YOU CANNOT INSTALL '$(echo '"'$arg'"' | tr a-z A-Z)' WITHOUT INSTALLING "UNZIP"!'; return 0
-						fi
-					fi
-					if grep -q 'zsync /opt' ./$arg; then
-						if ! command -v zsync 1>/dev/null; then
-							optzsync=$(grep -F 'if test -f /opt/$APP/*.zsync' ./$arg | wc -l)
-							if [ $optzsync == 0 ]; then
-								echo ' 💀 ERROR: YOU CANNOT INSTALL '$(echo '"'$arg'"' | tr a-z A-Z)' WITHOUT INSTALLING "ZSYNC"!'; return 0
-							else
-								echo -e ' ⚠️ WARNING: '$(echo '"'$arg'"' | tr a-z A-Z)' MAY NOT BE UPDATABLE WITHOUT "ZSYNC".\n\n THE APP WILL STILL BE INSTALLED, BUT INSTALLING "ZSYNC" IS RECOMMENDED!\n'
-							fi
-						fi
-					fi
-
-					# If you use the --latest option, add the "latest" flag to scripts that point to a very common "release"
-					if grep -q 'force-latest' $AMPATH/.cache/install-args; then
-						sed -i 's#releases -O -#releases/latest -O -#g' ./$arg
-					fi
+web_get() {
```

Comment:
See https://github.com/ivan-hc/AM/pull/538/files#r1588605326

Also, being this function already implemented in APP-MANAGER, it will be called also into the modules. You don't need to put it here too.

Reply:
Note yet exported from APP_MANAGER...